configuration/index.md

```diff
@@ -44,98 +47,132 @@ Every interface has it's own strong points.
+For Things and Items, both methods can be used **side-by-side**, e.g. a Thing can be discovered and created by Paper UI and the Items that link to that very same Thing (that Thing's Channels, actually) can be manually defined in a `.items` configuration file. 
+Configuration done in files will be used (and they become visible and even changeable in Paper UI) if no Thing/Item of the same name was already created in PaperUI, but you can NOT create a Thing or Item using files and then use Paper UI to redefine or change it. Those changes will not be accepted.
+Configuration done in PaperUI will be used and permanently stored in the internal database if no Thing/Item of the same name already exists in files, but those additions or changes will not be copied into any `.things` file.
+openHAB settings as defined in `services/addons.cfg` and `services/runtime.cfg` will override any settings made in PaperUI's Configuration/System pane.
+
+## Recommendations for New Users
+
+All interface options are supposed to work, still some of the more advanced settings might work better in one or the other setup.
+You have the full freedom of choice. On the other hand, this can be very confusing at times, particularly for new users.
+To help starters in narrowing down choices, here is some guidance that might also help to avoid some future pitfalls:
+
+* Use Paper UI to setup and maintain the openHAB base system. Use it to define _which_ addons and bindings you want to install and to assign _basic, static_ configuration (such as the device name of a ZWave stick or the IP address of a KNX gateway).
+* Use Paper UI to manage configuration of each and every openHAB 2 add-on and binding.
+* Use file configuration or Paper UI to manage configuration of openHAB 1 add-ons and bindings.
+* Use Paper UI or habmin to manage ZWave things, but use configuration files to manage ZWave items.<br/>
+  There is a rationale to this: to use a GUI is comfortable for one-time actions, but you get any repetitive task easier and faster done using a text editor (e.g. search-and-replace names and parameters).
+  _Things_ can be autodiscovered so you don't have to enter all of them manually. Once initially configured, their configuration is not changing much over time.
+  On the other hand, you will keep changing _items_ rather often.
+  If you are new to openHAB, you will e.g. keep changing all of your item names as you keep learning what is a naming scheme you like best to work with in your rules. And once you are a pro, you will want to keep using files anyway. You can configure items via Paper UI, too, and you can use HABmin as well for both but remember once you use files, this will override any UI-made changes on next openHAB startup.
+* Use the editor of your choice to edit [sitemaps]({{base}}/configuration/sitemaps.html). Start with the demo sitemap as a template.
+* Use VS code extensions to [edit rules, items and sitemap files.]({{base}}/configuration/editors.html).
```

Comment:
`{{base}}` isn't valid anymore - this will work, but only because a script will replace it for compatibility purposes. For new content, better to directly use: `[edit rules, items and sitemap files.](/docs/configuration/editors.html)`.

Reply:
Or simply [edit rules, items and sitemap files.](editors.html)